instrumentor/patch/root.go

```diff
@@ -39,6 +39,9 @@ const (
 	nodeInitContainerName        = "copy-nodejs-agent"
 	javaInitContainerName        = "copy-java-agent"
 	dotnetInitContainerName      = "copy-dotnet-agent"
+	easyConnectVersion           = "v1.0.9"
+	resourceAttrEnv              = "OTEL_RESOURCE_ATTRIBUTES"
+	resourceAttr                 = "easy.connect.version=%s"
```

Comment:
Will the dots be replaced with a dash at some point? Because in the readme it's with dashes, so just making sure that it's aligned :)

Reply:
not is will be with a dot. I will fix the readme